tools/wpt/tests/test_wpt.py

```diff
@@ -27,19 +65,21 @@ def test_help():
 
 
 @pytest.mark.slow
-@pytest.mark.system_dependent
 @pytest.mark.remote_network
-def test_run_firefox():
+def test_run_firefox(manifest_dir):
     # TODO: It seems like there's a bug in argparse that makes this argument order required
     # should try to work around that
+    if is_port_8000_in_use():
+        pytest.skip("port 8000 already in use")
```

Comment:
OK, I guess, but probably people aren't going to notice that they didn't run all the tests.